src/main/java/no/unit/nva/doi/transformer/CrossRefConverter.java

```diff
@@ -82,7 +82,7 @@ public Publication toPublication(CrossRefDocument document,
                 .withFileSet(createFilseSet())
                 .withEntityDescription(new EntityDescription.Builder()
                     .withContributors(toContributors(document.getAuthor()))
-                    .withDate(extractDate(document).orElse(null))
+                    .withDate(extractDate(document).orElse(new PublicationDate()))
```

Comment:
what if we do this to to the jackson serializer? mapper.setSerializationInclusion(Include.NON_NULL);

Reply:
I don' think we should do this. if we do only this, we will have an inconsistent behavior as fields are going to be initialized and some not.The alternative is to initialize a Publication object with empty objects and that we do not want either. We do not want to try saving empty objects in the database. We should fix the documentation so that the frontend knows  which types to use

Reply:
Yes, I believe you are right. We need to be able to identify missing data properly.